client/my-sites/sidebar/sidebar.jsx

```diff
@@ -650,50 +654,54 @@ module.exports = React.createClass( {
 					</ul>
 				</SidebarMenu>
 
-				{ vip ?
-				<SidebarMenu>
-					<SidebarHeading>VIP</SidebarHeading>
-					<ul>
-						{ this.vip() }
-						{ this.vipDeploys() }
-						{ this.vipBilling() }
-						{ this.vipSupport() }
-						{ this.vipBackups() }
-						{ this.vipLogs() }
-					</ul>
-				</SidebarMenu>
-				: null }
-
-				{ publish ?
-				<SidebarMenu>
-					<SidebarHeading>{ this.translate( 'Publish' ) }</SidebarHeading>
-					{ this.publish() }
-				</SidebarMenu>
-				: null }
-
-				{ appearance ?
-				<SidebarMenu>
-					<SidebarHeading>{ this.translate( 'Personalize' ) }</SidebarHeading>
-					<ul>
-						{ this.themes() }
-						{ this.menus() }
-					</ul>
-				</SidebarMenu>
-				: null }
-
-				{ configuration ?
-				<SidebarMenu>
-					<SidebarHeading>{ this.translate( 'Configure' ) }</SidebarHeading>
-					<ul>
-						{ this.sharing() }
-						{ this.users() }
-						{ this.plugins() }
-						{ this.upgrades() }
-						{ this.siteSettings() }
-						{ this.wpAdmin() }
-					</ul>
-				</SidebarMenu>
-				: null }
+				{ vip
```

Comment:
You could also update the code in this `render()` function to use the following shorter approach:

```javascript
{ vip &&
  <SidebarMenu>
    <SidebarHeading>VIP</SidebarHeading>
    <ul>
      { this.vip() }
      { this.vipDeploys() }
      { this.vipBilling() }
      { this.vipSupport() }
      { this.vipBackups() }
      { this.vipLogs() }
    </ul>
  </SidebarMenu>
}
```